cclient/serialization/json/helpers.c

```diff
@@ -83,6 +85,40 @@ retcode_t utarray_uint64_to_json_array(UT_array const* const ut,
   return RC_OK;
 }
 
+retcode_t neighbor_info_utarray_to_json_array(UT_array const* const ut,
```

Comment:
it should be a private method, move to `cclient/serialization/json/get_neighbors.c`